bin/p2-dsctl/main.go

```diff
@@ -70,14 +79,20 @@ func main() {
 
 	switch cmd {
 	case CmdCreate:
-		az := pc_fields.AvailabilityZone(*createAZ)
+		var selector labels.Selector
```

Comment:
How hard would it be to allow a raw selector to be expressed during the create operation? This tool is not very useful to Square without such an option.